type-definitions/most.d.ts

```diff
@@ -391,3 +391,23 @@ export function recoverWith<A, B>(p: (a: B) => Stream<A>, s: Stream<A>): Stream<
 export function throwError(e: Error): Stream<any>;
 
 export function multicast<A>(s: Stream<A>): Stream<A>;
+
+declare const defaultScheduler: Scheduler;
+export { defaultScheduler }
+
+export class PropagateTask<T> implements Task {
+	protected _run: (time: number, value: T, sink: Sink<T>) => any
+	protected value: T;
+	protected sink: Sink<T>;
+	protected active: boolean;
```

Comment:
Is is possible to omit _run and sink from the public type definition?  Hmmm, maybe it's ok to leave sink.  What do you think?